s3/replication/README.md

```diff
@@ -29,6 +29,12 @@ Run basic tests for each module. Clean, Build and test.
 make test
 ```
 
+## Jenkins build & test script
```

Comment:
Let title only say Build and test script.

In desc, mention this will be used for jenkins as well as developers to run full build and test.

Reply:
done